src/index.ts

```diff
@@ -56,6 +56,7 @@ const _onParentMessage = async ({ origin, data }: MessageEvent) => {
   switch (data.messageId) {
     case FromSafeMessages.ON_SAFE_INFO: {
       _logMessageFromSafe(origin, FromSafeMessages.ON_SAFE_INFO);
+      console.log(data.data);
```

Comment:
Is this good practice? Could te user of the sdk implement this in the listener?

Reply:
I think the best practice it to add a debug or verbose flag. we can implement this in the future